backend/routes/index.ts

```diff
@@ -7,9 +7,11 @@ const router = express.Router();
 
 router.get("/users", UserController.getUsers);
 router.get("/locations/:id", LocationsController.getLocationById);
+router.get("/locations/search/:zoom/:center/:bounds", LocationsController.getLocationsByZoom);
```

Comment:
Maybe, it's better to use req.query instead of req.params, as it's much easier on frontend to change query, rather than url (but it's just a remark, so it's up to you, change it or not).

Reply:
changed method to post, so it could be scaled in the future